01.general/15.dns-filtering/docs.en.md

```diff
@@ -0,0 +1,80 @@
+
+<a id="what-is-dns"></a>
+### What is DNS?
+
+DNS stands for "Domain name system", and its purpose is to translate websites' names into something browsers can understand, i.e. IP addresses. Thus, each time you go to a website, your browser sends a request to a special server (DNS server). That server looks at the requested domain name and replies with a corresponding IP address. Very schematically it can be represented like this:
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/DNS_filtering/how_dns_works_en.png" style="border: 1px solid #efefef; padding: 2px;" />
+
+The same applies, of course, to all apps and programs that send any web requests, not just browsers.
+
+<a id="dns-filtering"></a>
+### How does DNS filtering work?
+
+When you use one of the AdGuard apps that supports DNS filtering, it acts as a buffer between your device and the DNS server. All DNS requests that your browsers or apps are about to send first get processed by AdGuard. If you're using the default DNS server provided by your ISP, your DNS traffic is likely not encrypted and vulnerable to snooping and hijacking. AdGuard will encrypt all your DNS requests before they leave your device, so that no malefactor could get access to their contents. On top of that, AdGuard can identify requests to ad, tracking, and/or adult domains and redirect them to a "blackhole" instead of forwarding them to the DNS server. More on that [later](#dns-blocklists).
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/DNS_filtering/how_dns_filtering_works_en.png" style="border: 1px solid #efefef; padding: 2px;" />
+
+DNS filtering is a powerful tool and it's supported by all major AdGuard apps: [AdGuard for Windows](https://adguard.com/en/adguard-windows/overview.html), [AdGuard for Mac](https://adguard.com/en/adguard-mac/overview.html), [AdGuard for Android](https://adguard.com/en/adguard-android/overview.html) and [AdGuard for iOS](https://adguard.com/en/adguard-ios/overview.html).
+
+DNS filtering can be broken down into two separate functions: to encrypt and reroute DNS traffic to DNS servers, and to block some domains locally by applying DNS blocklists.
+
+<a id="dns-servers"></a>
+#### DNS servers
+
+There are thousands of DNS servers to choose from, and they are all unique in their properties and purposes. Most simply return the IP address of the requested domain, but some have additional functions: they block ad, tracking, adult domains and so on. Nowadays all major DNS servers employ one of the reliable encryption protocols, or several: DNS-over-HTTPS, DNS-over-TLS. AdGuard also provides a [DNS service](https://adguard.com/en/adguard-dns/overview.html), and it was the world's first to offer the very new and promising [DNS-over-QUIC](https://adguard.com/en/blog/dns-over-quic.html) encryption protocol. AdGuard has different servers for different goals. This diagram illustrates how AdGuard blocking servers work:
```

Comment:
one of the reliable encryption protocols, or several –> one or several reliable encryption protocols? / one of the reliable encryption protocols, or several ones